src/cml.js

```diff
@@ -21,17 +23,16 @@ const repo_from_origin = () => {
   return strip_auth(uri);
 };
 
+const isCI = () => {
+  return GITHUB_REPOSITORY || CI_PROJECT_URL || BITBUCKET_REPO_UUID;
```

Comment:
Often just `CI` is a generic env flag used for this sort of thing.

```suggestion
  return CI || GITHUB_REPOSITORY || CI_PROJECT_URL || BITBUCKET_REPO_UUID;
```

Reply:
🤔 I have not seen them in the ENV vars of GH, BB or GL